recipes/musclex/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+channel_sources:
+  - conda-forge,fastai,defaults
```

Comment:
We only want packages that depend on conda-forge in conda-forge. Please add any missing dependencies to conda-forge first.

Reply:
Hi, 

 i need to use opencv-python-headless, which is not available on conda-forge, which is why I used an alternative channel. Is there a solution to address that? 

Regards, 
Tristan Leduc

Reply:
Yes, we need a variant of that in conda-forge. See the opencv-feedstock for discussions/progress on it.